### ISSUE TITLE
New submissions for Thu, 11 Nov 21

### ISSUE THREAD

**Issue by imjunwei** [imjunwei/ocr-daily-paper#64]
## Keyword: Text Detection
There is no result 
## Keyword: Text Recognition
There is no result 
### Handwritten Digit Recognition Using Improved Bounding Box Recognition  Technique
 - **Authors:** Arkaprabha Basu, M. Sathya
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2111.05483
 - **Pdf link:** https://arxiv.org/pdf/2111.05483
 - **Abstract**
 The project comes with the technique of OCR (Optical Character Recognition) which includes various research sides of computer science. The project is to take a picture of a character and process it up to recognize the image of that character like a human brain recognize the various digits. The project contains the deep idea of the Image Processing techniques and the big research area of machine learning and the building block of the machine learning called Neural Network. There are two different parts of the project. Training part comes with the idea of to train a child by giving various sets of similar characters but not the totally same and to say them the output of this is this. Like this idea one has to train the newly built neural network with so many characters. This part contains some new algorithm which is self-created and upgraded as the project need. The testing part contains the testing of a new dataset .This part always comes after the part of the training .At first one has to teach the child how to recognize the character .Then one has to take the test whether he has given right answer or not. If not, one has to train him harder by giving new dataset and new entries. Just like that one has to test the algorithm also. There are many parts of statistical modeling and optimization techniques which come into the project requiring a lot of modeling concept of statistics like optimizer technique and filtering process, that how the mathematics and prediction behind that filtering or the algorithms comes after or which result one actually needs to and ultimately for the prediction of a predictive model creation. Machine learning algorithm is built by concepts of prediction and programming.
### BagBERT: BERT-based bagging-stacking for multi-topic classification
 - **Authors:** Loïc Rakotoson, Charles Letaillieur, Sylvain Massip, Fréjus Laleye
 - **Subjects:** Computation and Language (cs.CL); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2111.05808
 - **Pdf link:** https://arxiv.org/pdf/2111.05808
 - **Abstract**
 This paper describes our submission on the COVID-19 literature annotation task at Biocreative VII. We proposed an approach that exploits the knowledge of the globally non-optimal weights, usually rejected, to build a rich representation of each label. Our proposed approach consists of two stages: (1) A bagging of various initializations of the training data that features weakly trained weights, (2) A stacking of heterogeneous vocabulary models based on BERT and RoBERTa Embeddings. The aggregation of these weak insights performs better than a classical globally efficient model. The purpose is the distillation of the richness of knowledge to a simpler and lighter model. Our system obtains an Instance-based F1 of 92.96 and a Label-based micro-F1 of 91.35.
### What Makes Online Communities 'Better'? Measuring Values, Consensus, and  Conflict across Thousands of Subreddits
 - **Authors:** Galen Weld, Amy X. Zhang, Tim Althoff
 - **Subjects:** Social and Information Networks (cs.SI); Computers and Society (cs.CY); Human-Computer Interaction (cs.HC)
 - **Arxiv link:** https://arxiv.org/abs/2111.05835
 - **Pdf link:** https://arxiv.org/pdf/2111.05835
 - **Abstract**
 Making online social communities 'better' is a challenging undertaking, as online communities are extraordinarily varied in their size, topical focus, and governance. As such, what is valued by one community may not be valued by another. In this work, we measure community values through a survey of 2,769 reddit users in 2,151 unique subreddits, the largest survey of community values to date. Through a combination of survey responses and a quantitative analysis of publicly available reddit data, we characterize how these values vary within and across communities. We find that there is 47.4% more disagreement over how safe communities are than disagreement over other aspects of communities' current state, that longstanding communities place 30.1% more emphasis on trustworthiness than newer communities, and that recently joined redditors perceive their communities more positively than more senior redditors. We find that community moderators want their communities to be 56.7% less democratic than non-moderator community members. Accurate and scalable modeling of community values enables research and governance which is tuned to each community's different values. To this end, we demonstrate that a small number of automatically quantifiable features captures a substantial amount of the variation in values between communities with a ROC AUC of 0.667 on a binary classification task. However, significant variation remains, and modeling community values remains an important topic for future work. We make our models and data public to inform community design and governance.
## Keyword: Handwriting
There is no result 
## Keyword: Scene Text
There is no result 
## Keyword: Text Segmentation
There is no result 
## Keyword: TextSpotter
There is no result 
## Keyword: Text Spotting
There is no result
